.github/actions/release-image/action.yml

```diff
@@ -58,6 +58,13 @@ runs:
         archs: ${{ inputs.melangeArchs }}
         template: ${{ inputs.melangeTemplate }}
 
+    # If publishing to Chainguard registry (cgr.dev), setup auth
+    - id: cgrauth1
+      if: contains(inputs.apkoBaseTag, 'cgr.dev/') && inputs.chainguardIdentity != ''
```

Comment:
We should just do this whenever there's an identity IMO

Reply:
we can add back if needed, but for some matrix legs in other repos, may not be doing anything related to cgr.dev